locale/index.js

```diff
@@ -5,7 +5,20 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import extensionTranslations from 'Extensions/translations';
+import isPlainObject from 'lodash/isPlainObject';
+import defaultsDeep from 'lodash/defaultsDeep';
+
 // eslint-disable-next-line import/no-dynamic-require
-const templateTranslations = require(`./${process.env.LANG}.json`);
+const themeTranslations = require(`./${process.env.LANG}.json`);
+let extension = {};
+
```

Comment:
Please move this to a helper function as part of the pwa-common. Also add a comment for this code block